evalml/automl/engine/engine_base.py

```diff
@@ -96,7 +96,7 @@ def train_pipeline(pipeline, X, y, optimize_thresholds, objective):
 
         Returns:
             pipeline (PipelineBase) - trained pipeline.
-        """
+        """        
```

Comment:
Am I reading this right?  Is white space being added here?

Reply:
Yeah, unsure why this was here. Fixed!